### DISCUSSION TITLE
What power cable should be used between GreaseWeazle and a 3.5" PC floppy?

### DISCUSSION THREAD

**Discussion by jolsen71** [keirf/greaseweazle#493] [Q&A] [answered]
What power cable should be used between GreaseWeazle and a 3.5" PC floppy? And does anyone know where one can be ordered? 

It seems like maybe a Berg connector on the floppy side, and is it 1.25 pitch 4 pin on the GreaseWeazle side? if not, what are the proper connections? This might be a candidate for the FAQ/Wiki?

Thanks

**Comment by keirf** [comment, marked as answer]
Same both sides: Berg. Although 4-pin 2.54mm pitch may do. Get one of those sloppy splitter cables with two Bergs and a Molex. Ignore the Molex, or cut and splice it out.
